### DISCUSSION TITLE
Provide the ability to open a bim file stored in a vpax file.

### DISCUSSION THREAD

**Discussion by corycundy** [TabularEditor/TabularEditor3#795] [Ideas]
It would be nice to have the ability to open a bim file that is stored in a vpax file if there is one.

This comes in handy when doing quick analysis on a vpax file that is sent to you. It would be handy to open TE3, point to the file and have both the VertiPaq Analyzer info import and the model if stored.

One option could be when importing a vpax file, detect if there is a bim file stored within and prompt to open it as well. If a solution is already open, it could prompt as it would if you opened a new model when a solution is open.

Cory

**Comment by mlonsk** [comment]
Hi @corycundy 
Sorry for the very long delay in getting a response back to you. 

This is possible in the newest version of Tabular Editor 3, as you can see in the bellow GIF.

![VPAX Model Open](https://github.com/TabularEditor/TabularEditor3/assets/11939114/01191646-b227-47cd-b7f7-d98f86064b2c)
